test/oauth-authorization-test.js

```diff
@@ -478,21 +540,18 @@ suite.addBatch({
                         "and we submit the authentication form": {
                             topic: function(browser) {
                                 var cb = this.callback;
-                                browser.fill("username", "alice", function(err) {
-                                    if (err) {
-                                        cb(err);
-                                    } else {
-                                        browser.fill("password", "white*rabbit", function(err) {
-                                            if (err) {
-                                                cb(err);
-                                            } else {
-                                                browser.pressButton("#authenticate", function(err) {
-                                                    cb(err, browser);
-                                                });
-                                            }
-                                        });
+
+                                browser
+                                    .fill("username", "alice")
+                                    .fill("password", "white*rabbit");
+                                Step(
+                                    function(){
+                                        browser.pressButton("#authenticate",this);
+                                    },
+                                    function(){
```

Comment:
Watch your whitespace on the preceding four lines - specifically you need a space between `)` and `{`, and a space between `,` and `this`.